bot.py

```diff
@@ -268,7 +271,7 @@ def karma_check(post, pauthor):
     if user.link_karma + user.comment_karma < 5:
         print("\tKARMA KO")
         slay(post, KARMARM)
-        post.report("Low karma OP")
+        reddit.subreddit(SUBREDDIT).message("upmo - karma", SPAMCHECK + post.link)
```

Comment:
Is the `reddit.subreddit(SUBREDDIT).*` part needed in full, or could [this part](https://github.com/unixporn/upmo/blob/d93cdd1724ddd39f8b50f53ec52759a747fcac39/bot.py#L414) be used and just do `subreddit.*`?

Also, check your PEP8 validation, some of your lines are a bit too long

Reply:
If I'm not mistaken, this is the only function using it, I think it's lighter to do `r.subreddit(SUBREDDIT)` rather than passing `subreddit` as an argument.